src/components/shared/MarkerTooltipContents.js

```diff
@@ -122,6 +122,20 @@ function _makePhaseTimesArray(
   return array;
 }
 
+function _dataPrioReplace(str: number) {
+  if (str === -20) {
+    return 'Priority: Highest (-20)';
+  } else if (str === -10) {
+    return 'Priority: High (-10)';
+  } else if (str === 10) {
+    return 'Priority: Low (10)';
```

Comment:
<img width="218" alt="image" src="https://user-images.githubusercontent.com/1588648/44488360-2da7c700-a61e-11e8-9700-99aea593e28f.png">

This results in "Priority: Priority", I think it should just be:

```
return 'Low (10)';
```